Clicker/ViewControllers/PollsDateViewController+Extension.swift

```diff
@@ -73,25 +81,30 @@ extension PollsDateViewController: PollsDateSectionControllerDelegate {
 }
 
 extension PollsDateViewController: PollBuilderViewControllerDelegate {
-    
-    func startPoll(text: String, type: QuestionType, options: [String], state: PollState, correctAnswer: String?, shouldPopViewController: Bool) {
+    func startPoll(text: String, type: QuestionType, options: [String], state: PollState, answerChoices: [PollResult], correctAnswer: String?, shouldPopViewController: Bool) {
+        
         createPollButton.isUserInteractionEnabled = false
         createPollButton.isHidden = true
-        
-        let correct = correctAnswer ?? ""
-        
-        // EMIT START QUESTION
-        let socketQuestion: [String: Any] = [
-            RequestKeys.textKey: text,
-            RequestKeys.typeKey: type.descriptionForServer,
-            RequestKeys.optionsKey: options,
-            RequestKeys.sharedKey: state == .shared,
-            RequestKeys.correctAnswerKey: correct
+
+        let newPoll = Poll(text: text, answerChoices: answerChoices, type: type, correctAnswer: correctAnswer, userAnswers: [:], state: .live)
+
+        let answerChoicesDict = answerChoices.compactMap { $0.dictionary }
+
+        var correct = ""
+        if let correctAnswer = correctAnswer {
+            correct = correctAnswer
+        }
```

Comment:
ok i'm not sure, but can this be:

`let correct = correctAnswer ?? ""`

@mindylou @kevinchan159 help

Reply:
yes sir